tests/cases/folder_test.py

```diff
@@ -38,7 +38,7 @@ def tearDownModule():
 
 class FolderTestCase(base.TestCase):
     def setUp(self):
-        events.unbindAll()
```

Comment:
@zachmullen Is this safe, or do we need to more selectively remove certain events here? The code was added in your commit https://github.com/girder/girder/commit/d2052b2be2767fe39d8d23f2cc023d93c9e097d3#diff-2e7bf9531d7542fbed2103c654d83bc5R40 .